src/fitnesse/resources/bootstrap/templates/skeleton.vm

```diff
@@ -61,5 +61,18 @@
   </footer>
   #end
 
+  <script>
+   function adjustPadding() {
+    var navHeight = document.querySelector('nav').offsetHeight;
```

Comment:
Just to be sure I suggest you check whether `document.querySelector('nav')` gives a result before trying to use it.

Reply:
Added a simple check. If a nav Element does not exist then there won't be an exception in the Console now.